.github/workflows/changelog.yml

```diff
@@ -0,0 +1,8 @@
+on: pull_request
```

Comment:
Do you think we should have a `name` attribute?

```
name: changelog
on: pull_request

jobs:
```

Reply:
What's the attribute for?